mono/eglib/glib.h

```diff
@@ -1148,4 +1186,22 @@ glong     g_utf8_pointer_to_offset (const gchar *str, const gchar *pos);
  
 G_END_DECLS
 
-#endif
+// For each allocator; i.e. returning gpointer that needs to be cast.
+// Macros do not recurse, so naming function and macro the same is ok.
+// However these are also already macros.
+#undef g_malloc
+#undef g_realloc
+#undef g_malloc0
+#undef g_calloc
+#undef g_try_malloc
+#undef g_try_realloc
+#undef g_memdup
+#define g_malloc(x) (g_cast (monoeg_malloc (x)))
```

Comment:
Just declare that at https://github.com/mono/mono/pull/10027/files#diff-b8c6c2ba344450884373fb00f33685adR186 instead, and rename the `g_malloc` function to `g_malloc_internal` if it collides.

Reply:
And then breakpoints on `g_malloc` no longer work.

Reply:
Er, breakpoint is `monoeg_malloc`. I guess just break on `malloc`, though sometimes the extra layer is nice, i.e. to break only on mono's calls.